backend/src/emissions/tests/test_plantrip.py

```diff
@@ -21,42 +21,169 @@
+    variables = {"transportationMode": transportation_mode, "fuelType": fuel_type, "roundtrip": roundtrip}
+    response = requests.post(GRAPHQL_URL, json={"query": query, "variables": variables}, headers=headers)
+    logger.warning(response.content)
+    assert response.status_code == 200
+    data = response.json()
+    logger.warning(data)
+    assert data["data"]["planTrip"]["success"]
+
 
+@pytest.mark.parametrize('transportation_mode', ['Plane'])
+@pytest.mark.parametrize('seating_class', ['average', 'Economy class', 'Premium economy class', 'Business class', 'First class'])
+@pytest.mark.parametrize('roundtrip', [True, False])
+def test_plan_trip_plane(transportation_mode, seating_class, roundtrip):
+    """Test whether trip planner throws error for different parameter combinations"""
+    query = """
+        mutation planTrip ($transportationMode: String!, $seatingClass: String!, $roundtrip: Boolean!) {
+            planTrip (input: {
+                transportationMode: $transportationMode
+                seatingClass: $seatingClass
+                roundtrip: $roundtrip
+                distance: 500
+            }) {
+                success
+                message
+                co2e
+            }
+        }
+    """
+
+    headers = {
+        "Content-Type": "application/json",
+    }
+    variables = {"transportationMode": transportation_mode, "seatingClass": seating_class, "roundtrip": roundtrip}
+    response = requests.post(GRAPHQL_URL, json={"query": query, "variables": variables}, headers=headers)
+    logger.warning(response.content)
+    assert response.status_code == 200
+    data = response.json()
+    logger.warning(data)
+    assert data["data"]["planTrip"]["success"]
+
+
+@pytest.mark.parametrize('transportation_mode', ['Ferry'])
+@pytest.mark.parametrize('seating_class', ['average', 'Foot passenger', 'Car passenger'])
+@pytest.mark.parametrize('roundtrip', [True, False])
+def test_plan_trip_ferry(transportation_mode, seating_class, roundtrip):
+    """Test whether trip planner throws error for different parameter combinations"""
+    query = """
+        mutation planTrip ($transportationMode: String!, $seatingClass: String!, $roundtrip: Boolean!) {
+            planTrip (input: {
+                transportationMode: $transportationMode
+                seatingClass: $seatingClass
+                roundtrip: $roundtrip
+            }) {
+                success
+                message
+                co2e
+            }
+        }
+    """
+
+    headers = {
+        "Content-Type": "application/json",
+    }
+    variables = {"transportationMode": transportation_mode, "seatingClass": seating_class, "roundtrip": roundtrip}
+    response = requests.post(GRAPHQL_URL, json={"query": query, "variables": variables}, headers=headers)
+    logger.warning(response.content)
+    assert response.status_code == 200
+    data = response.json()
+    logger.warning(data)
+    assert data["data"]["planTrip"]["success"]
```

Comment:
Are we only testing the success scenarios?

Reply:
for now yes, but we should definitely also test error message at some point as well. The co2emission calculations are tested in the co2calculator package.